apps/mobile/src/components/Search/Community/CommunitySearchResult.tsx

```diff
@@ -6,15 +6,14 @@ import { MentionType } from 'src/hooks/useMentionableMessage';
 import { CommunityProfilePicture } from '~/components/ProfilePicture/CommunityProfilePicture';
 import { CommunitySearchResultFragment$key } from '~/generated/CommunitySearchResultFragment.graphql';
 import { MainTabStackNavigatorProp } from '~/navigation/types';
-import { noop } from '~/shared/utils/noop';
 
 import { SearchResult } from '../SearchResult';
 
 type Props = {
   communityRef: CommunitySearchResultFragment$key;
   onSelect?: (item: MentionType) => void;
```

Comment:
is there a place where onSelect isn't passed in? maybe it should be required

Reply:
@kaitoo1 
Yup! On normal search screen we don't pass `onSelect`. 

The default value of for `onSelect` will be navigate to info screen (User/Community/Gallery)